src/functions-reference/covariance_matrix_distributions.Rmd

```diff
@@ -137,25 +141,77 @@ Increment target log probability density with `inv_wishart_lupdf(W | nu, Sigma)`
 \index{{\tt \bfseries inv\_wishart\_lpdf }!{\tt (matrix W \textbar\ real nu, matrix Sigma): real}|hyperpage}
 
 `real` **`inv_wishart_lpdf`**`(matrix W | real nu, matrix Sigma)`<br>\newline
-The log of the inverse Wishart density for symmetric and
-positive-definite matrix W given degrees of freedom nu and symmetric
-and positive-definite scale matrix Sigma
+Return the log of the inverse Wishart density for symmetric and
+positive-definite matrix `W` given degrees of freedom `nu` and symmetric
+and positive-definite scale matrix `Sigma`.
 `r since("2.12")`
 
 <!-- real; inv_wishart_lupdf; (matrix W | real nu, matrix Sigma); -->
 \index{{\tt \bfseries inv\_wishart\_lupdf }!{\tt (matrix W \textbar\ real nu, matrix Sigma): real}|hyperpage}
 
 `real` **`inv_wishart_lupdf`**`(matrix W | real nu, matrix Sigma)`<br>\newline
-The log of the inverse Wishart density for symmetric and
-positive-definite matrix W given degrees of freedom nu and symmetric
-and positive-definite scale matrix Sigma dropping constant additive terms
+Return the log of the inverse Wishart density for symmetric and
+positive-definite matrix `W` given degrees of freedom `nu` and symmetric
+and positive-definite scale matrix `Sigma` dropping constant additive terms.
 `r since("2.25")`
 
 <!-- matrix; inv_wishart_rng; (real nu, matrix Sigma); -->
 \index{{\tt \bfseries inv\_wishart\_rng }!{\tt (real nu, matrix Sigma): matrix}|hyperpage}
 
 `matrix` **`inv_wishart_rng`**`(real nu, matrix Sigma)`<br>\newline
-Generate an inverse Wishart variate with degrees of freedom nu and
-symmetric and positive-definite scale matrix Sigma; may only be used
-in transformed data and generated quantities blocks
+Generate an inverse Wishart variate with degrees of freedom `nu` and
+symmetric and positive-definite scale matrix `Sigma`; may only be used
+in transformed data and generated quantities blocks.
 `r since("2.0")`
+
+
+## Inverse Wishart distribution, Cholesky Parameterization {#inverse-wishart-cholesky-distribution}
+
+### Probability density function
+
+If $K \in \mathbb{N}$, $\nu \in (K-1, \infty)$, and $L_S \in \mathbb{R}^{K \times K}$ is 
+lower triangular with positive diagonal elements then $L_S L_S^{\top}$ is positive definite. 
+A random lower triangular matrix with positive diagonal $L_W \in \mathbb{R}^{K \times K}$ 
+such that $W = L_WL_W^{\top}$ is distributed  
+\[
```

Comment:
Sorry to be such a stickler on the math, but the equation here is still not quite right.  I'd lead with this:

if W ~ InvWishart(nu, S), then L_W ~ InvWishartCholesky(nu, L_S).  

But then if you want to define the density of L_W, you need the Jacobian, so that'd be:

InvWishartCholesky(L_W | nu, L_S) = WishartCholesky(L_W * L_W' | nu, L_S * L_S') * | J |

where J is the Jacobian of the inverse transform f^-1(L_W) = L_W * L_W'.    

I wouldn't try to go down the rabbit hole of properly defining Jacobians for matrix functions (serialize the matrix args to an array) given that you define the log absolute Jacobian.